apps/infra/scripts/sst-event-log-security.test.mjs

```
@@ -0,0 +1,200 @@
+    wrapper.kill('SIGTERM')
+    const result = await wrapperExit
+
+    assert.equal(await fileExists(eventLog), false, 'SIGTERM must not leave the Pulumi event log behind')
+    assert.deepEqual(result, { code: 143, signal: null })
+  } finally {
+    if (wrapper.exitCode === null && wrapper.signalCode === null) wrapper.kill('SIGKILL')
+    if (Number.isSafeInteger(childPid)) {
+      try {
+        process.kill(childPid, 'SIGKILL')
+      } catch (error) {
+        if (error.code !== 'ESRCH') throw error
```

Biome (2.5.5)

[error] Unsafe usage of 'throw'.

(lint/correctness/noUnsafeFinally)